ext/php8/ddtrace.c

```diff
@@ -748,6 +808,22 @@ static PHP_FUNCTION(additional_trace_meta) {
     zval_copy_ctor(return_value);
 }
 
+/* {{{ proto string DDTrace\add_global_tag(string $key, string $value) */
+static PHP_FUNCTION(add_global_tag) {
+    UNUSED(execute_data);
+
+    zend_string *key, *val;
+    if (zend_parse_parameters(ZEND_NUM_ARGS(), "SS", &key, &val) == FAILURE) {
+        RETURN_THROWS();
```

Comment:
We should not throw. We can log a message in debug mode and proceed the execution.
In general, we never throw exceptions, so users' applications can keep working even if with tracing not perfectly working